shared/common-adapters/icon.desktop.js

```diff
@@ -32,20 +33,29 @@ class Icon extends PureComponent<void, Exact<Props>, void> {
     const fontSizeHint = shared.fontSize(iconType)
 
     if (isFontIcon) {
-      const cleanStyle = {...this.props.style}
+      const cleanStyle = {
+        fontFamily: 'kb',
+        speak: 'none',
+        fontStyle: 'normal',
+        fontWeight: 'normal',
+        fontVariant: 'normal',
+        textTransform: 'none',
+        lineHeight: 1,
+        WebkitFontSmoothing: 'antialiased',
+        ...this.props.style}
       // We have to blow these styles away else FontIcon gets confused and will overwrite what it calculates
       delete cleanStyle.color
       delete cleanStyle.hoverColor
 
       return <FontIcon
         title={this.props.hint}
         style={{...globalStyles.noSelect, ...styles.icon, ...fontSizeHint, ...cleanStyle, ...(this.props.onClick ? globalStyles.clickable : {})}}
-        className={`icon-kb-${iconType}${this.props.className && ' ' + this.props.className || ''}`}
+        className={`${this.props.className && ' ' + this.props.className || ''}`}
         color={color}
         hoverColor={this.props.onClick ? hoverColor : null}
         onMouseEnter={this.props.onMouseEnter}
         onMouseLeave={this.props.onMouseLeave}
-        onClick={this.props.onClick} />
+        onClick={this.props.onClick}>{String.fromCharCode(iconMeta[iconType].charCode || 0)}</FontIcon>
```

Comment:
Should we have this display an obviously broken icon? how about: ![](https://pbs.twimg.com/profile_images/425274582581264384/X3QXBN8C.jpeg) ;)